tcpdf.php

```diff
@@ -4103,7 +4103,7 @@ public function GetStringWidth($s, $fontname='', $fontstyle='', $fontsize=0, $ge
 
 	/**
 	 * Returns the string length of an array of chars in user unit or an array of characters widths. A font must be selected.<br>
-	 * @param string $sa The array of chars whose total length is to be computed
+	 * @param array $sa The array of chars whose total length is to be computed
```

Comment:
Maybe copy return of `$sa = TCPDF_FONTS::UTF8ArrToLatin1Arr($sa);` ? since the function seems to do a copy

Reply:
@williamdes not in every case. See `@param boolean $getarray if true returns an array of characters widths, if false returns the total length.`
So the return should be `@return float[]|float` imho.

Reply:
I've changed this in 6e988d94de25b1e92c0d26adffe27a87dbc497d7